### PR TITLE
Adds copyright year as a template variable

### LINKED ISSUE #1088: Add template variable for copyright year
There are two reasons for this:

- It should be possible to override the copyright year, because the time of project generation is not necessarily identical to the date of initial publication. For example, an existing project can be ported to use this project template.
- Tools that update projects with changes from their project templates should not update the copyright year. If the copyright year is stored in a template variable, this will just work. If it is embedded in the license template, it won't.

Note that copyright notices need only contain the initial date of publication (if at all). There is no need to keep an up-to-date range of years. See [TIL: You Can Stop Updating Copyright Attribution Years](https://hynek.me/til/copyright-years/).

### PR DESCRIPTION
**Pull request by jasonwashburn**
Resolves #1088 
Adds copyright year as a template variable set during project creation.

### CODE REVIEW

**Comment by jasonwashburn** [comment]
Looks like the Check Documentation workflow failed because of a temporary error at https://badgen.net/badge/status/alpha/d8624d . Link seems to be working for me this morning?

File: {{cookiecutter.project_name}}/docs/conf.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
 """Sphinx configuration."""
-from datetime import datetime
```
Comment (cjolowicz) [inline {{cookiecutter.project_name}}/docs/conf.py:3]: Let's remove the double blank line now that there are no imports.

**Review by cjolowicz — COMMENTED** [review]
Thank you, this looks good! See comments.

**Comment by jasonwashburn** [comment]
Done 👍

**Review by cjolowicz — APPROVED** [review]

**Pull request merged**